packages/commonwealth/server/controllers/server_threads_methods/create_thread.ts

```diff
@@ -137,59 +135,39 @@ export async function __createThread(
     canvas_session: canvasSession,
     canvas_hash: canvasHash,
     discord_meta: discordMeta,
+    topic_id: +topicId,
   };
 
+  if (
+    community &&
+    (community.type === ChainType.Token ||
```

Comment:
Wait doesn't this mean gating isn't enabled for Cosmos chains since `Community.type === chain` for Cosmos?

Reply:
Good catch -- and that's been present the entire time. Will push a fix.

Reply:
Please resolve if new commit fixes.

Reply:
Fixes this instance but I think I see the same thing in `create_thread_comment.ts` maybe we should create a ticket to review all instances.

Reply:
Resolving for now - will ticket.

Reply:
I should have addressed that one as well @timolegros